Dockerfile

```diff
@@ -1,15 +1,30 @@
-# docker build . -t sncli
+# docker build . [--build-arg editor_packages=neovim] -t sncli
 # docker run --rm -it -v /tmp:/tmp -v "$HOME/.sncli/:/root/.sncli/" -v "$HOME/.snclirc:/root/.snclirc" sncli
 FROM python:3.8-buster
 
-RUN pip3 install pipenv
+ARG editor_packages="neovim"
```

Comment:
I would prefer if the default were more conservative, such as `vim` (I'm not sure what the equivalent package on debian is).  This would also be consistent with the default editor if EDITOR isn't set.